python/ray/serve/model_wrappers.py

```diff
@@ -1,14 +1,15 @@
 from collections import defaultdict
 from typing import Any, Callable, Dict, List, Optional, Tuple, Type, Union
+
 import numpy as np
 
+import ray
+from ray import serve
 from ray._private.utils import import_attr
 from ray.air.checkpoint import Checkpoint
-from ray.air.predictor import Predictor
 from ray.serve.drivers import HTTPAdapterFn, SimpleSchemaIngress
-import ray
-from ray import serve
 from ray.serve.utils import require_packages
+from ray.train.predictor import Predictor
```

Comment:
Put this under type checking?

Reply:
We can't do that, right? `_load_predictor_cls` has an isinstance check. Unless we import it in there

Reply:
Ah, will leave this up to @simon-mo, but we can also do something like this

```
try:
    from ray.train.predictor import Predictor
except ImportError:
    Predictor = None
```

and then in `_load_predictor_cls` do `if Predictor is not None and issubclass(predictor, Predictor)`